portfolio/src/main/webapp/script.js

```diff
@@ -88,30 +88,52 @@ function getComments() {
 getComments();
 
 /**
- * Checks login status of user, links to login/logout page, hides comment form
- * if user is logged out.
+ * Fetches login status data from /login and updates comment-form div in DOM
+ *     depending on login status. Depending on status, DOM is updated to hide
+ *     the form and prompt the user to login or out.
  */
-function checkLoginStatus() {
+function updateCommentForm() {
   fetch('/login')
   .then(response => response.json())
   .then((status) => {
-    const baseUrl = window.location.origin;
-    let url = new URL(status[1], baseUrl);
-
     let commentForm = document.getElementById('comment-form');
-    let para = document.createElement("p");
     
-    if (status[0] === "True") {
-      para.innerHTML = "<p>Logout <a href=\"" + url + "\">here</a>.</p>";
+    if (status['isLoggedIn'] === true) {
```

Comment:
Does the style guide say anything about `status['isLoggedIn']` vs `status.isLoggedIn`?

Reply:
I haven't been able to find anything about that in the style guide, but some [other](https://codeburst.io/javascript-quickie-dot-notation-vs-bracket-notation-333641c0f781) [sites](https://stackoverflow.com/questions/4968406/javascript-property-access-dot-notation-vs-brackets) suggest that dot notation is easier to read, so I'll switch other to that.